### DISCUSSION TITLE
【求助】绳网怎么进

### DISCUSSION THREAD

**Discussion by silver-fox217** [share121/inter-knot#2317] [其他]
兄弟们，绳网怎么进啊
搞了半天找不到教程，现在挺急的
有兄弟知道的请马上回复

**Comment by anything12121** [comment]
在线等教程

**Comment by SouthThink** [comment]
我也不会，在线等教程
<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by shiyuanniexijiaoxue** [comment]
我也，在线等教程（

**Comment by yanlei2009** [comment]
我也不会啊
<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by ling333920** [comment]
#1898 包你满意<img src="https://bbs-static.miyoushe.com/static/2023/10/07/a74bef45413382fafe41111d15d299b0_273291246745481614.png" alt="undefined" width="75" >

**Comment by 6beihongcha** [comment]
我不道啊

**Comment by liuying5201314** [comment]
我也不知道，在线等<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="undefined" width="75" >

**Comment by xiaomwm** [comment]
在线等教程 急急急

**Comment by nd304** [comment]
我不道啊
![Uploading img_-993186449_1721580585722.jpeg…]()

**Comment by Yith123** [comment]
你按一下F5就能进了

**Comment by dengyunzhe** [comment]
Alt+F<sub>4</sub>
